apps/ideaspace/src/components/modules/WorkshoppingPage/CommentsForm/CommentForm.js

```diff
@@ -68,14 +83,24 @@ function CommentForm(props) {
               placeholder="What are your thoughts? (max 250 characters)"
               value={props.handleTextChange}
               onChange={handleTextChange}
+<<<<<<< Updated upstream
```

Comment:
this looks like an unresolved conflict marker... it looks to me like we can delete it along with all of the code on line 87 to 92 so we can keep our `CommentBox`